client/blocks/product-selector/index.jsx

```diff
@@ -102,64 +114,30 @@ export class ProductSelector extends Component {
 		return map( products, product => {
 			const selectedProductSlug = this.state[ this.getStateKey( product.id, intervalType ) ];
 			const productObject = storeProducts[ selectedProductSlug ];
+			const stateKey = this.getStateKey( product.id, intervalType );
 
 			return (
-				<Fragment key={ 'product-' + product.id }>
-					<ProductCard
-						key={ product.id }
-						title={ product.title }
-						billingTimeFrame={ this.getBillingTimeFrameLabel() }
-						fullPrice={ productObject.cost }
-						description={ product.description }
-						currencyCode={ currencyCode }
-						purchase={ this.getPurchaseByProduct( product ) }
-						subtitle={ this.getSubtitleByProduct( product ) }
+				<ProductCard
+					key={ product.id }
+					title={ product.title }
+					billingTimeFrame={ this.getBillingTimeFrameLabel() }
+					fullPrice={ productObject.cost }
+					description={ <p>{ product.description }</p> }
```

Comment:
Noting that this feels off. This makes it obvious that the `<ProductCard />` should create a wrapping `<p>` automagically when we pass it just a string. We can fix in a subsequent PR.

Reply:
So I've introduced this change in one of the previous PRs but now I agree it doesn't seem right. My assumption was that you'll get more flexibility if the description is not wrapped up in a `<p>` tag. However, the way we use `description` always requires the paragraph tag anyway.

I'll change it in the next PR.

Reply:
I agree about the flexibility, but not requiring a wrapping `<p>` while supporting react elements makes it even more flexible 😉

Reply:
What I mean is that once we bake in the `<p>` tag inside the component and stop requiring it as part of the `description` field, we lose a flexibility of having e.g. multiple paragraphs in there (since we cannot nest `<p>` tags).